src/Spline.cpp

```diff
@@ -200,10 +200,25 @@ double base::geometry::SplineBase::getCurveLength(double relative_resolution) co
     return length;
 }
 
-double SplineBase::getUnitParameter()
```

Comment:
Just checked the commit log, sorry. There is kind of a relationship in "regular" curves (which is what SISL tries to produce).

The issue you're facing here is that by removing it you make the situation worse by using geometric_resolution as a parameter iteration, which is even most wrong.